src/GitHub.App/ViewModels/PullRequestCreationViewModel.cs

```diff
@@ -96,8 +96,18 @@ public class PullRequestCreationViewModel : BaseViewModel, IPullRequestCreationV
                         notifications.ShowError(error?.Message ?? ex.Message);
                         return Observable.Empty<IPullRequestModel>();
                     }));
+
+            CancelCommand.Subscribe(_ => ClearForm());
+           
+        }
+
+        private void ClearForm()
+        {
+            this.PRTitle = string.Empty;
+            this.Description = string.Empty;
         }
-        
```

Comment:
The other thing that needs to be cleared is the TargetBranch. Setting it to `TargetBranch = Branches.FirstOrDefault(b => b.Name == "master");` should do it.